CMakeLists.txt

```diff
@@ -789,17 +789,14 @@ endif()
 include(CoverageReport)
 
 # HLSL Change begin - Set default distribution value.
-if (NOT CMAKE_CONFIGURATION_TYPES)
-  set(LLVM_DISTRIBUTION_COMPONENTS "dxc;dxcompiler;dxc-headers" CACHE STRING "")
-endif ()
+set(LLVM_DISTRIBUTION_COMPONENTS "dxc;dxcompiler;dxc-headers" CACHE STRING "")
 # HLSL Change end - Set default distribution value.
```

Comment:
It's a little awkward to set the default here rather than in some dxc-specific place, but I guess it's okay for now